src/main/groovy/io/gatling/gradle/GatlingPluginExtension.groovy

```diff
@@ -2,6 +2,51 @@ package io.gatling.gradle
 
 class GatlingPluginExtension implements JvmConfigurable {
 
+    private static final String API_TOKEN_PROPERTY = "gatling.enterprise.apiToken"
+    private static final String API_TOKEN_ENV = "GATLING_ENTERPRISE_API_TOKEN"
+
+    static class Enterprise {
+        private String apiToken
+        private UUID packageId
+        private URL url = new URL("https://cloud.gatling.io/api/public")
+
+        def url(String url) {
+            this.url = new URL(url)
+        }
+
+        def packageId(String packageId) {
+            this.packageId = UUID.fromString(packageId)
+        }
+
+        def apiToken(String apiToken) {
```

Comment:
better stick to the MOJO notation: `setApiToken`

Reply:
Done that way so that configuration can be specified as below:
````
gatling {
  enterprise {
    apiToken '2j-VeP3m.ePPs5OO8KMkFwHKOP_wpXDJnFBDqStPe9vbqWUUqAArIo.ItRfY5Kgug'
    packageId '5950272d-f783-4981-9de4-c977f46b9cb6'
    url 'http://localhost:10542/api/public'
  }
}
````
instead of
````
gatling {
  enterprise {
    setApiToken '2j-VeP3m.ePPs5OO8KMkFwHKOP_wpXDJnFBDqStPe9vbqWUUqAArIo.ItRfY5Kgug'
    setPackageId '5950272d-f783-4981-9de4-c977f46b9cb6'
    setUrl 'http://localhost:10542/api/public'
  }
}
````

Reply:
OK